assets/css/protocols/quiz1.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sourceRoot":"","sources":["../../sass/protocols/quiz1.scss","../../sass/components/icons.scss"],"names":[],"mappings":";AAEA;AAIE;EACE;EACH;;AAGC;EACE;;AAGF;EACE;;AAGF;EACE;EACA;EACA;EACA;EACA;;AAGF;EACE;EACA;EACA;EACA;EACA;;AAGF;AAAA;EAEE;;AAGF;EACE;;AAGF;EACE;;AAGF;EC9CD;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EDwCG;;AAEA;EACE;;AAIJ;ECvDD;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EDiDG;;AAEA;EACE;;AAIJ;EChED;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;ED0DG;;AAEA;EACE","file":"quiz1.css"}
```

Comment:
I think this file was not supposed to be included.

Reply:
Just to clarify, which files are to be included in git after changes in styles or new .scss-files? I noticed that some of the `.map` are there, so just to be sure.

Reply:
All `.map` discribe where the source lines can be found for minimised files. E.g. `main.css` is unreadable as is, but `main.css.map` tells where every line is from, thus when you open browser dev tools, you get to read the actual sources and not just unreadable files.

In short, every compressed/minified/generated *.css or *.js file should have a *.map file too. You can notice that the *.css file includes a comment at the end of the file, which points to the *.map file (see `/*# sourceMappingURL=main.css.map */` in the `main.css`).

Reply:
That clears it up well, thanks :)